osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,34 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows for the use of any existing network that relies on an existing VPC.
+
+.Prerequisites
+* You have a configured VPC. Use the following documentation for guidance on how to configure the VPC's components, such as:
```

Comment:
An assembly can have a prereqs section if you want, which could make sense for the "AWS account" part. But it would look like 

```
[//specify ID]
== Prerequisites
```

You are allowed to give a summary of the things that you'll accomplish in the assembly, but it probably wouldn't be in a prereqs section like this